### PR TITLE
alt text to images in news items for accessibility, prettier'd this time

### PR DESCRIPTION
**Pull request by gmerritt**
Alt text added to screenshot images included with three news items to satisfy WCAG 2.1 level A 1.1.1 Non-text Content ("Image missing a text alternative").

Files formatted with prettier.

### CODE REVIEW

**Comment by nwalters512** [comment]
I'll review in the morning! Could you explain what's different in this PR vs #9364?

**Comment by gmerritt** [comment]
@nwalters512, the previous one failed CI because my changes weren't up to the prettier spec. In this one, the three html files have been reformatted with prettier; watching now for CI to complete.

**Comment by nwalters512** [comment]
If you need to make changes in the future, just push new commits to the same branch; no need to open a brand new PR to respond to comments or fix a failing CI check.

File: apps/prairielearn/src/news_items/043_units_element/index.html [hunk 1]
```diff
@@ -35,22 +35,42 @@
   submissions like <code>100 cm</code> or <code>1000 mm</code> are also accepted.
 </p>
 
-<img class="border rounded" src="pl-units-input.png" width="800px" />
+<img
+  class="border rounded"
+  src="pl-units-input.png"
+  alt="Screenshot of a problem requring units"
```
Comment (nwalters512) [inline apps/prairielearn/src/news_items/043_units_element/index.html:41]:
```suggestion
  alt="Screenshot of a problem requiring units"
```

**Review by nwalters512 — COMMENTED** [review]
One spelling fix and this will be good to go!

**Review by nwalters512 — APPROVED** [review]

**Pull request merged**